lastools-el7/fpm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+version=170414
```

Comment:
At the moment getting the version is a manual step. In a year's time is anyone going to remember how we got this version number (minus any comments of course). Better would be to:

1. unzip the binary
1. Run `laszip --version` (capturing the version number)

This has the benefit of being self-documenting, we know where the version comes from and keeps it in sync